app/services/queue_health_check.rb

```diff
@@ -0,0 +1,14 @@
+class QueueHealthCheck
+  def perform
+    health_check_jobs.each do |job|
+      $redis.set(job.downcase, 'dead')
+      Object.const_get("HealthCheck::#{job}QueueJob").perform_later
```

Comment:
Yeah, I tried that when creating this, actually! I just tried again and had same result - I was getting `undefined method set for Nil::NilClass` -- restarted my server, and it worked! Guess I should have realized that it was not working because rails was not aware that I'd added the `CheckerJob` class without restarting my server. I blame Spring!

Anyway, great suggestion. Updated at e6529e4